api/config/rolepermission.php

```diff
@@ -692,6 +694,15 @@
             'en' => 'View any basic government employee profile',
             'fr' => 'Voir tout profil de base d\'employé du gouvernement',
         ],
+
+        'view-any-poolActivityLog' => [
+            'en' => 'View Any Pool Activity Log',
+            'fr' => 'Afficher le journal des activités de la piscine',
+        ],
+        'view-team-poolActivityLog' => [
+            'en' => 'View team Pool Activity Log',
+            'fr' => 'Afficher le journal d\'activité du pool d\'équipes',
```

Comment:
I don't think we are using `pool` in French for pool in English, are we @NienkeBr? The term `bassin` seems to be scattered throughout the codebase.